braze-templates/4-sms-capture-modal/index.html

```diff
@@ -0,0 +1,657 @@
+                        Updates!</button>
+                </div>
+            </div>
+            <div class="modal__content" id="thankyou">
+                <button data-button-id="close" class="modal__cross modal__cross-web">
+                    <svg width="34" height="34" viewBox="0 0 34 34" fill="none" xmlns="http://www.w3.org/2000/svg">
+                        <path fill-rule="evenodd" clip-rule="evenodd"
+                            d="M14.4175 16.8419L8.51467 10.9391C7.8452 10.2696 7.8452 9.18422 8.51467 8.51475C9.18414 7.84528 10.2696 7.84528 10.939 8.51475L16.8418 14.4176L22.7446 8.51475C23.4141 7.84528 24.4995 7.84528 25.169 8.51475C25.8385 9.18422 25.8385 10.2696 25.169 10.9391L19.2662 16.8419L25.169 22.7447C25.8385 23.4142 25.8385 24.4996 25.169 25.1691C24.4995 25.8386 23.4141 25.8386 22.7446 25.1691L16.8418 19.2663L10.939 25.1691C10.2696 25.8386 9.18414 25.8386 8.51467 25.1691C7.8452 24.4996 7.8452 23.4142 8.51467 22.7447L14.4175 16.8419Z"
+                            fill="#A8B3B8" />
+                    </svg>
+                </button>
+                <h1 class="modal__title">Thanks for signing up!</h1>
+                <!-- <h2 class="modal__sub-title">Sub head maybe here that is not too long</h2> -->
+                <p class="modal__text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod.</p>
+                <div class="modal__button-wrap">
+                    <a class="modal__btn modal__btn--main" data-button-id="learn more button">Learn
+                        More!</a>
+                </div>
+            </div>
+        </div>
+    </div>
+
+    <script>
+        // button click tracking
+        document.querySelectorAll('[data-button-id]').forEach(function (node) {
+            node.addEventListener('click', function () {
+                appboyBridge.logClick(node.dataset.buttonId);
+                appboyBridge.closeMessage();
+                appboyBridge.requestImmediateDataFlush();
+            });
+        });
+    </script>
+    <script>
+        var input = document.getElementById("phone");
+        var error = document.getElementById("error");
+        var optin = document.getElementById("optin");
+        var submit = document.getElementById("submit");
+        var thankyou = document.getElementById("thankyou");
+        var form = document.getElementById('form');
+        var iti = window.intlTelInput(input, {
+            nationalMode: true,
+            hiddenInput: 'full',
+            utilsScript: "https://appboy-images.com/appboy/communication/assets/code_assets/files/60ba5392f9088d62254859d1/original.js?1622823826"
+        });
+        window.intlTelInputGlobals.loadUtils("https://appboy-images.com/appboy/communication/assets/code_assets/files/60ba5392f9088d62254859d1/original.js?1622823826");
+        // prevent enter key submissions
+        $('html').bind('keypress', function (e) {
```

Comment:
Is this the only place where jQuery is used? If so, can we just get rid of it? It would be trivial to replace this with `addEventListener`

Reply:
the phone number library is dependent on jquery :(